project/dexter-core/src/resource/dexter.em

```diff
@@ -588,12 +597,16 @@ macro saveModifiedFunctionList(){
 			
 		}else if(symbolLocation.Type == "Function" || symbolLocation.Type == "Method"){
 			functionName = SymbolLeafName(symbolLocation);

```

Comment:
Macro Language doesn't have "continue" statement.
This if block "jumps over" one iteration in the loop.
If symbolLocation is "", the symbolLocation handling (like symbolLocation.Type) throw an exception.